tests/test_kitsune.py

```diff
@@ -211,6 +216,18 @@ def test_fetch_empty(self):
 
         self.assertEqual(len(questions), 0)
 
+    @httpretty.activate
```

Comment:
In this tests we access the page (69) which generates the 500 error that it is managed inside the backend, and no questions are returned. Then, the next page (70) is retrieved which does not exists and generates a 404 (not found) error which is captured by the assert. Then, the total number of pages is checked, which must be 0.